takwimu/takwimu_ui/src/components/Navigation/DropDowns.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
```

Comment:
`DropDowns` sucks... may `Insights`, or `ActionableInsights`, etc.

Reply:
I think its best dropdown because the next developer will probably search for dropdown or drawer rather than insight

Reply:
Aaight...